### ISSUE TITLE
Magic Gradients controls

### ISSUE THREAD

**Issue by filipoff2** [mgierlasinski/MagicGradients#113]
## Magic Controls
- [Magic Button](https://github.com/mgierlasinski/MagicGradients/issues/118)
- Magic Page
- [Magic Progress](https://github.com/mgierlasinski/MagicGradients/issues/149)
- Animated buttons
    - metallic style
    - egg style
    - with shader

**Comment by mgierlasinski** [comment]
@filipoff2 Good idea, specs about each control would be nice :) We have first candidate in progress: #118 

When we have more controls, we can move them to separate Nuget, something like:
- MagicGradients.Controls
- MagicGradients.UI
- other better name
